guides/gatherers.md

```diff
@@ -1575,3 +1577,119 @@ Example output (in Rhai):
 // hacluster_has_default_password
 true;
 ```
+
+<span id="ini_files"></span>
+
+### ini_files@v1
+
+**Argument required**: yes.
+
+This gatherer fetches the content from a configuration file in INI format. The configuration file is specified as argument, choosen from a list of allowed files.
+Currently whitelisted files are:
```

Comment:
niptick: If you want this to be in a new line, you need to add a new line before the phrase